lib/beaker-puppet/install_utils/foss_utils.rb

```diff
@@ -761,9 +761,9 @@ def install_puppet_from_dmg_on( hosts, opts )
                 raise "You need to specify versions for OSX host\n eg. install_puppet({:version => '3.6.2',:facter_version => '2.1.0',:hiera_version  => '1.3.4',})"
               end
 
-              on host, "curl -O #{opts[:mac_download_url]}/puppet-#{puppet_ver}.dmg"
-              on host, "curl -O #{opts[:mac_download_url]}/facter-#{facter_ver}.dmg"
-              on host, "curl -O #{opts[:mac_download_url]}/hiera-#{hiera_ver}.dmg"
+              on host, "curl -LO #{opts[:mac_download_url]}/puppet-#{puppet_ver}.dmg"
+              on host, "curl -LO #{opts[:mac_download_url]}/facter-#{facter_ver}.dmg"
+              on host, "curl -LO #{opts[:mac_download_url]}/hiera-#{hiera_ver}.dmg"
```

Comment:
in case youn want to update it: I think long options are always better. open source libs are read by a lot of people. since you already touch those lines, you could also switch to long options.

Reply:
I updated to add the long option for `location`, but now that I think about it were you suggesting I also replace `-O` for `--remote-name` ?

Reply:
> I updated to add the long option for `location`, but now that I think about it were you suggesting I also replace `-O` for `--remote-name` ?

I think it's a good idea to do so :+1: